scylla/src/transport/session_builder.rs

```diff
@@ -605,15 +605,15 @@ impl SessionBuilder {
     /// # use scylla::transport::session::{AddressTranslator, TranslationError};
     ///
```

Comment:
It looks like there is an unnecessary space at the beginning of the box with the example code (above `let mut translation_rules = ...`):

![image](https://user-images.githubusercontent.com/4950936/193861497-c20e415f-e65c-4cf4-9c25-9ad77034d407.png)